### PR TITLE
Added reports UI pages and routes

### LINKED ISSUE #31: Reports Pages
Add the Monthly, Quarterly, and Year-End Report pages to the UI

### PR DESCRIPTION
**Pull request by stickzman**
Added reports pages. Fixes #31

### CODE REVIEW

File: routes.php [hunk 1]
```diff
@@ -15,4 +15,11 @@
...
-$router->add('/locations', 'locations/locations.php', 'Locations');
\ No newline at end of file
+$router->add('/locations', 'locations/locations.php', 'Locations');
+
+#Reports
+$router->add('/quarterly-reports', 'reports/quarterly.php', 'Quarterly Reports');
+$router->add('/year-end-reports', 'reports/half_year.php', 'Year-End Reports');
+$router->add('/monthly-reports', 'reports/monthly_report.php', 'Monthly Reports');
+$router->add('/custom-reports', 'reports/custom_reports.php', 'Custom Report');
+$router->add('/custom_reports_table.php', 'reports/custom_reports_table.php', 'Custom Report');
```
Comment (blackjack26) [inline routes.php]: Change the URL from `/custom_reports_table.php` to `/custom-reports-table`.

File: views/reports/custom_reports.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                        <input type="checkbox" name="age" id="age-2" value="March">
+                        65+
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <!-- Button -->
+            <div class="form-group">
+                <div class="col-md-4">
+                    <a href="custom_reports_table.php" class="btn btn-primary">Generate Report</a>
```
Comment (blackjack26) [inline views/reports/custom_reports.php]: Change `custom_reports_table.php` to `/custom-reports-table`. All page navigation is through routes.

**Comment by ghost** [comment]
Looks like we need to add a standard comment header. The format is in the [wiki](https://github.com/Capping-CPCA/Web-App/wiki/Adding-New-Pages)

**Review by ghost — CHANGES_REQUESTED** [review]
Each new page needs a header comment, standard found in the wiki

File: views/reports/custom_reports.php (hunk 2 above)
Comment (blackjack26) [reply views/reports/custom_reports.php]: Also don't use the class `btn-primary`, use `cpca` instead.

File: views/reports/half_year.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                        </select>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <div class="row pb-5">
+            <div class="col"></div>
+            <div class="col-centered">
+                <p>
+                    <a class="btn btn-primary btn-large" href="#">Generate Report</a>
```
Comment (blackjack26) [inline views/reports/half_year.php]: `cpca` instead of `btn-primary`

File: views/reports/monthly_report.php [hunk 4]
```diff
@@ -0,0 +1,1234 @@
...
+                        </select>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <div class="row pb-5">
+            <div class="col"></div>
+            <div class="col-centered">
+                <p>
+                    <a class="btn btn-primary btn-large" href="#">Generate Report</a>
```
Comment (blackjack26) [inline views/reports/monthly_report.php]: `cpca` instead of `btn-primary`

File: views/reports/half_year.php [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                </p>
+            </div>
+            <div class="col"></div>
+        </div>
+    </div>
+    <div class="container py-3">
+        <h1 class="text-center">
+        Semi-Annual 2017</h3>
```
Comment (blackjack26) [inline views/reports/half_year.php]: Tag mismatch. This tag is opened with an `<h1>` but closed with an `</h3>`

File: views/reports/quarterly.php [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+                <p>
+                    <a class="btn btn-primary btn-large" href="#">Generate Report</a>
+                </p>
+            </div>
+            <div class="col"></div>
+        </div>
+    </div>
+    <div class="container py-3">
+        <h1 class="text-center">
+        Q1 2017</h3>
```
Comment (blackjack26) [inline views/reports/quarterly.php]: Tag mismatch. This tag is opened with an `<h1>` but closed with an `</h3>`.

File: views/reports/custom_reports.php [hunk 7]
```diff
@@ -0,0 +1,162 @@
+﻿<?php
```
Comment (blackjack26) [inline views/reports/custom_reports.php]: This file is saved using the encoding `UTF-8-BOM` which could have occurred if you were using notepad to write the file. Change this file encoding to `UTF-8` (the `UTF-8-BOM` adds a whitespace to the top of this page)

File: views/reports/custom_reports_table.php [hunk 8]
```diff
@@ -0,0 +1,48 @@
+﻿<?php
```
Comment (blackjack26) [inline views/reports/custom_reports_table.php]: This file is saved using the encoding `UTF-8-BOM` which could have occurred if you were using notepad to write the file. Change this file encoding to `UTF-8` (the `UTF-8-BOM` adds a whitespace to the top of this page)

**Review by blackjack26 — CHANGES_REQUESTED** [review]

**Comment by blackjack26** [comment]
Also I would suggest adding the `table-striped` CSS class to all the tables. Makes them easier to read / look at. Also maybe even `table-bordered` or some way to make sure each table is separately distinguishable.

**Review by ghost — APPROVED** [review]

**Comment by blackjack26** [comment]
Looks good to me. One last thing, I'll be releasing this merge with a version tag of `0.1.4`. Since it's only the interface part it'll be a small version release. Once the actual functionality is put it, it'll be a larger version update. So in your comments put:
```php
/**
 * @version 0.1.4
 * @since 0.1.4
 */
```

Also rebase to develop to make it a single commit (squash).

**Review by blackjack26 — APPROVED** [review]

**Pull request merged**